backend/db/schemas/hacknews_db.sql

```diff
@@ -0,0 +1,39 @@
+-- 
+DROP TABLE IF EXISTS `users`;
+CREATE TABLE IF NOT EXISTS `users` (
+  `id`                        INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `login_id`                  VARCHAR(255)     NOT NULL UNIQUE,
+  `password`                  VARCHAR(255)     NOT NULL,
+  `auth_token`                TEXT             NULL,
+  `name`                      VARCHAR(255)     NOT NULL,
+  `avatar_url`                TEXT             NULL,
+  `created_at`                TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
what's the range of `TIMESTAMP`, does it have year 2038 problem?

Reply:
Yes, it's going to occur 2038 problem.
Int or Datetime will be better.